### ISSUE TITLE
Android upgrade/downgrade subscription native dialog is not refreshed within one session

### ISSUE THREAD

**Issue by jpudysz** [dooboolab/react-native-iap#1027]
### Version of react-native-iap
4.4.9
### Version of react-native
0.62.2
### Platforms you faced the error (IOS or Android or both?)
Android
### Expected behavior
Native dialog with information about subscription upgrade/downgrade
### Actual behavior
When I buy subscription A I can upgrade or downgrade it with subscription B, but there is one edge case which won't work (please follow steps to reproduce).

If I don't close the app after successful payment, Android can't detect that I've already bought other subscription. Only after app restart it's starting correct flow.

Is there a way to refresh module? I tried to close/init connection after purchase but it didn't affect the native dialog.

### Tested environment (Emulator? Real Device?)
Real Device Samsung J5
### Steps to reproduce the behavior

Scenario 1 (Incorrect):
1. Buy subscription A
2. Without closing the app try to buy Subscription B
3. Native dialog is present without detecting that you already bought other subscription so there is no information about how it will affect your current subscription

Scenario 2 (correct):
1. Buy subscription A
2. Close the app
3. Try to buy Subscription B
4. Native dialog is shown with the upgrade/downgrade calculations

Additional notes:
I'm using `Deferred` proration mode
I call `consumeAllItemsAndroid` after module initialization
I always call `finishTransaction` after each payment

**Comment by hyochan** [comment]
I am still unsure why this is happening in the `deferred` mode. I remember other modes worked ok. I also need more information about this.
